content/opt/rundeck-defaults/profile

```diff
@@ -34,7 +34,8 @@ RDECK_JVM_SETTINGS="${RDECK_JVM_SETTINGS:- -Xmx1024m -Xms256m -XX:MaxMetaspaceSi
 RDECK_TRUSTSTORE_FILE="${RDECK_TRUSTSTORE_FILE:-$RDECK_CONFIG/ssl/truststore}"
 RDECK_TRUSTSTORE_TYPE="${RDECK_TRUSTSTORE_TYPE:-jks}"
 JAAS_LOGIN="${JAAS_LOGIN:-true}"
-JAAS_CONF="${JAAS_CONF:-$RDECK_CONFIG/jaas-loginmodule.conf}"
+JAAS_CONF_FILE="${JAAS_LOGIN_CONF:-jaas-loginmodule.conf}"
```

Comment:
This should be JAAS_CONF_FILE="${JAAS_CONF_FILE:-jaas-loginmodule.conf}" so users can override the filename if needed, right?